module/channel/src/Infrastructure/Grid/ExportGridBuilder.php

```diff
@@ -42,6 +42,16 @@ public function build(GridConfigurationInterface $configuration, Language $langu
                         'export' => '{id}',
                     ],
                 ],
+                'download' => [
```

Comment:
This will add this action to every export and not every export will have file to download..